File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest02423.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest02423 {

    @GetMapping("/BenchmarkTest02423")
    public void BenchmarkTest02423(@RequestHeader("Host") String host, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String hostValue = host != null ? host : "";
        java.util.Map.Entry<String,String> kv = java.util.Map.entry(hostValue, "request");
        response.setHeader("X-Tuple-Context", kv.getValue());
        String data = kv.getKey();
        request.isUserInRole("ADMIN");
        response.getWriter().print("{\"role\":\"admin\"}");
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_02423.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest02423():
    multipart_value = request.form.get('multipart_field', '')
    data = f'{multipart_value:.200s}'
    data = bytearray(int(data) if str(data).isdigit() else 0)
    return jsonify({"result": "success"})
